dhall-json.cabal

```diff
@@ -30,10 +30,14 @@ Library
     Hs-Source-Dirs: src
     Build-Depends:
         base                 >= 4.8.0.0  && < 5   ,
+        bytestring                          < 0.11,
         aeson                >= 1.0.0.0  && < 1.4 ,
         dhall                >= 1.11.0   && < 1.12,
         text                 >= 0.11.1.0 && < 1.3 ,
-        unordered-containers                < 0.3
+        trifecta             >= 1.6      && < 1.8 ,
```

Comment:
I think you can now remove the `trifecta` dependency from the `dhall-to-json` and `dhall-to-yaml` executables

Reply:
Done.